spec/lib/puppet_litmus/serverspec_spec.rb

```diff
@@ -50,4 +50,24 @@ class DummyClass
       end
     end
   end
+
+  describe '.bolt_upload_file' do
+    let(:local) { '/tmp' }
+    let(:remote) { '/remote_tmp' }
+    let(:result) { ['status' => 'success', 'result' => { 'exit_code' => 0, 'stdout' => nil, 'stderr' => nil }] }
+    let(:inventory_hash) { Hash.new(0) }
```

Comment:
There is no test for if inventory_hash is nil, which it will be when, as above, TARGET_HOST is set to 'localhost'.